### ISSUE TITLE
Migrate jZebra to qz-print

### ISSUE THREAD

**Issue by bahoo** [five23/shipsync-community#24]
qz-print appears to be jZebra's spiritual successor. I recently did this for a client and would volunteer to re-integrate this into shipsync.

see:
https://code.google.com/p/jzebra/
https://github.com/qzindustries/qz-print

... unless someone's already done this, or there's a good reason not to. Anybody? Bueller?

**Comment by five23** [comment]
This is _definitely_ something that needs to be done...  I started tinkering with qz-print a few months ago but haven't had time to do much with it so any help would be much appreciated....     Should we put together some milestones and split the work up?

**Comment by bahoo** [comment]
Yeah, I like this idea. Thankfully the API hasn't changed much; it's just `qz.append64(label)` & print, instead of `jZebra.append(label)`, etc.. Do you want to take point on divvying up work?

Would be good to look at other options too, like a toggle for auto-printing labels once they're created. I've got a little `<iframe>` trick which might come in handy here.

**Comment by five23** [comment]
Yeah, I get a lot of emails from people having problems with the thermal printing functionality ... The setup isn't intuitive and if it isn't exactly right the labels don't print correctly..   Definitely utilizing some of the new functionality in qz print to autodetect and allow for on-the-fly options would be super neat.   

I'd suggest the first step would be to simply swap out jZebra with qz-print without adding or modifying anything, then work on pulling the logic out of the indexcontroller (where it really doesn't belong)..   Since you've already touched on this do you want to go ahead and do that, then I'll take a look and make a few milestones to expand the functionality?

**Comment by bahoo** [comment]
Cool, I'm good with that. I've got an exciting week coming up but I'll follow up once I've got something ready to show off. Cheers!
